shuup/core/templatetags/shuup_common.py

```diff
@@ -114,6 +114,9 @@ def datetime(value, kind="datetime", format="medium", tz=True):
 
     locale = get_current_babel_locale()
 
+    if value is None:
```

Comment:
Based on docsting this function does not allow None as a parameter. So add it or/and can you give some extra information why this is needed?